projects/step-frontend/src/lib/modules/plan-editor/components/plan-editor-actions/plan-editor-actions.component.ts

```diff
@@ -55,6 +56,7 @@ export class PlanEditorActionsComponent implements OnChanges {
     undo: 'Undo (Ctrl + Z)',
     redo: 'Redo (Ctrl + Y)',
     duplicate: 'Duplicate this plan',
+    showSource: `Show plan's YAML source`,
```

Comment:
```suggestion
    showSource: 'Show plan's YAML source',
```

Reply:
@neogucky  \` quotes are explicitly used, because string already contains a single quote inside in word `plan's`

Reply:
I missed that, but in that case I would prefer to escape the quote instead of using another quote style